cmd/experimental/kjobctl/pkg/builder/slurm_builder.go

```diff
@@ -571,22 +587,24 @@ func (b *slurmBuilder) buildInitEntrypointScript() (string, error) {
 }
 
 type slurmEntrypointScript struct {
-	EnvsPath               string
-	SbatchEnvFilename      string
-	SlurmEnvFilename       string
-	UnmaskFilenameFunction string
-	ChangeDir              string
-	BuildEntrypointCommand string
+	EnvsPath                string
+	SbatchEnvFilename       string
+	SlurmEnvFilename        string
+	UnmaskFilenameFunction  string
+	ChangeDir               string
+	BuildEntrypointCommand  string
+	WaitForFirstNodeTimeout int32
```

Comment:
```suggestion
	WaitForFirstNodeTimeoutSeconds int32
```
which is a convention to make the unit clear, or keep as time.Duration here.